### PR TITLE
Path planner setup

### PR DESCRIPTION
**Pull request by mxweII**
added the AutoBuilder in Swerve and registered the commands and paths that we have

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 1]
```diff
@@ -47,6 +54,31 @@ private Swerve() {
...
         swerveEstimator = new SwerveDrivePoseEstimator(Constants.Swerve.swerveKinematics, getGyroYaw(), getModulePositions(), getPose());
+
+         AutoBuilder.configureHolonomic(
+                this::getPose, // Robot pose supplier
+                this::setPose, // Method to reset odometry (will be called if your auto has a starting pose)
+                this::getRobotRelativeSpeeds, // ChassisSpeeds supplier. MUST BE ROBOT RELATIVE
+                this::driveRobotRelative, // Method that will drive the robot given ROBOT RELATIVE ChassisSpeeds
+                new HolonomicPathFollowerConfig( // HolonomicPathFollowerConfig, this should likely live in your Constants class
+                        new PIDConstants(5.0, 0.0, 0.0), // Translation PID constants
+                        new PIDConstants(5.0, 0.0, 0.0), // Rotation PID constants
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: I'm guessing these should match our existing swerve PID constants right? If so, we might as well use those constants here. Same with the max speed and base radius below.

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 2]
```diff
@@ -132,6 +164,12 @@ public void resetModulesToAbsolute(){
             mod.resetToAbsolute();
         }
     }
+    public ChassisSpeeds getRobotRelativeSpeeds(){
+        return ChassisSpeeds.fromRobotRelativeSpeeds(getRobotRelativeSpeeds(), getGyroYaw());
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: Is this recursive? If so, I'm guessing we need to rethink where we're getting ChassisSpeeds for this call.

File: src/main/java/frc/robot/subsystems/Swerve.java [hunk 3]
```diff
@@ -132,6 +164,12 @@ public void resetModulesToAbsolute(){
...
+    }
+    public void driveRobotRelative(ChassisSpeeds speed){
+        new ChassisSpeeds(speed.vxMetersPerSecond, speed.vyMetersPerSecond, speed.omegaRadiansPerSecond);
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/Swerve.java]: What is this method intended to do? It looks like it's just creating a new ChassisSpeeds object and not doing anything with it or returning anything.

**Review by notrandomness — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Swerve.java (hunk 3 above)
Comment (mxweII) [reply src/main/java/frc/robot/subsystems/Swerve.java]: I just updated these methods do these look right now?

**Review by notrandomness — APPROVED** [review]

**Pull request merged**